tests/addUserUI.spec.ts

```diff
@@ -63,16 +66,52 @@ test("Check 'Gender' field content on the 'Add User' page", async ({
 }) => {
   const genderField = page.locator('xpath=//*[@id="selectGender"]');
   await expect(genderField).toBeVisible;
-  // TODO: update locators which use Label
+
   // checking option 1 for gender input - Male
-  await genderField.selectOption({ label: GenderOptions[1] });
-  await expect(genderField).toHaveValue(GenderOptions.Male.toString());
+  await genderField.selectOption("1");
+  expect(await extractSelectedDisplayedValue(genderField)).toBe(
+    GenderOptions[1],
+  );
 
   // checking option 2 for gender input - Female
-  await genderField.selectOption({ label: GenderOptions[2] });
-  await expect(genderField).toHaveValue(GenderOptions.Female.toString());
+  await genderField.selectOption("2");
+  expect(await extractSelectedDisplayedValue(genderField)).toBe(
+    GenderOptions[2],
+  );
 
   // checking option 0 for gender input - Undefined
-  await genderField.selectOption({ label: GenderOptions[0] });
-  await expect(genderField).toHaveValue(GenderOptions.Undefined.toString());
+  await genderField.selectOption("0");
+  expect(await extractSelectedDisplayedValue(genderField)).toBe(
+    GenderOptions[0],
+  );
+});
+
+test("Verify Header content on the 'Add User' page", async ({ page }) => {
+  // getting first listitem of the header
+  let listitem = page.locator("xpath=//ul/li[position()<2]");
+  await expect(listitem).toBeVisible;
+  await expect(listitem).toHaveRole("listitem");
+  // checking the content of the child of first listitem of the header, which is a link
+  let child = listitem.locator("xpath=child::*");
+  await expect(child).toHaveRole("link");
+  await expect(child).toContainText("Home");
+  await expect(child).toHaveAttribute("href", "/");
+
+  // getting the next listitem of the header
+  listitem = listitem.locator("xpath=following-sibling::li[1]");
+  await expect(listitem).toBeVisible();
+  await expect(listitem).toHaveRole("listitem");
+  // checking the content of the child of second listitem, which is a link
+  child = listitem.locator("xpath=descendant::*");
+  await expect(child).toHaveRole("link");
+  await expect(child).toContainText("Add User");
+  await expect(child).toHaveAttribute("href", URLS.ADDUSER);
+
+  // checking the content of last listitem of the header
+  child = listitem
+    .locator("xpath=ancestor::ul")
```

Comment:
Do we need to use a double locator search? 
Can we use it inside one locator search?